### PR TITLE
Updates for OpenShift GitOps GA

### PR DESCRIPTION
**Pull request by sabre1041**
Updates for OpenShift GitOps GA

### CODE REVIEW

File: openshift/gitops/manifests/bootstrap/argocd-operator/base/argocd-subscription.yaml [hunk 1]
```diff
@@ -4,12 +4,12 @@ metadata:
...
   namespace: openshift-operators
 spec:
-  channel: preview
+  channel: stable
   installPlanApproval: Automatic
   name: openshift-gitops-operator
   source: redhat-operators
   sourceNamespace: openshift-marketplace
-  startingCSV: openshift-gitops-operator.v1.0.0
+  startingCSV: openshift-gitops-operator.v1.1.0
```
[the remark this answers is not in the document]
Comment (sabre1041) [reply openshift/gitops/manifests/bootstrap/argocd-operator/base/argocd-subscription.yaml:12]: Should we just remove?

[the remark this answers is not in the document]
Comment (nasx) [reply openshift/gitops/manifests/bootstrap/argocd-operator/base/argocd-subscription.yaml:12]: Yeah the operator should resolve the version. Stable channel starts at 1.1.0 anyway I believe, so probably safe to just remove.

[the remark this answers is not in the document]
Comment (sabre1041) [reply openshift/gitops/manifests/bootstrap/argocd-operator/base/argocd-subscription.yaml]: done

File: README.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ Next, modify `openshift/gitops/clusters/overlays/byo/bootstrap/redhat-portal-cre
 We are now ready to bootstrap the environment. To do this, run:
 
 ```shell
-kustomize build --load-restrictor=LoadRestrictionsNone openshift/gitops/clusters/overlays/byo/bootstrap/ | oc apply -f -
+kustomize build --load_restrictor=LoadRestrictionsNone openshift/gitops/clusters/overlays/byo/bootstrap/ | oc apply -f -
```
[the remark this answers is not in the document]
Comment (sabre1041) [reply README.md:82]: ```      --load_restrictor string   if set to 'LoadRestrictionsNone', local kustomizations may load files from outside their root. This does, however, break the relocatability of the kustomization. (default "LoadRestrictionsRootOnly")```

**Comment by nasx** [comment]
LGTM

**Pull request merged**